src/app/components/split-view/index.html

```diff
@@ -0,0 +1,239 @@
+            <p>Tree views can also be used if the items in the list are hierarchical. You may use different views in the workspace panel if the items at different levels of the tree are of different types.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-list-type-tree.png" />
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>While <a routerLink="/components/grid">grids</a> may technically be used, avoid them if possible. They do not scale well to the typically-small width of the navigation list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-list-type-grid.png" />
+            <div class="caption caption-warning sky-text-warning">Use caution when using grids as the list type as they do not perform well at narrow widths.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      <h3>List manipulation</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>If the user will need to search for a specific item in the list or need to show/hide items with a certain status, put a search input and/or status checkbox filter in the list panel.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-search-toggle.png" />
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Pre-filter the list to relevant items and pre-sort the list to the expected useful order when possible. If you can't predict how the user will need to use the list you may include a list toolbar so that users can filter, sort and search the list, or add new items to the list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-toolbar.png" />
+            <div class="caption caption-warning sky-text-warning">Use caution when using a full toolbar with a split view. Automatically filter and sort the list when possible.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Behavior and states">
+      <h3>Automatically advance to the next item</h3>
+        <div class="sky-text-block">
+          <p>If the user is likely to work through the list of items sequentially, automatically advance to the next item in the list when the user invokes the primary action on the record.</p>
+        </div>
+      <h3>Set focus in the workspace</h3>
```

Comment:
Is the spacing a bit off here?
![Screen Shot 2019-07-17 at 2 23 12 PM](https://user-images.githubusercontent.com/18354182/61400608-9cc58400-a89e-11e9-9046-11231bb8ce06.png)

Reply:
More or less thing the same thing as with my last comment. This text fits the first column. Usually the image appears beside it, but in your browser, the image doesn't fit in that column. I think we should go with this as is for now and revisit it later.